Editor/UI/JitInspectorView.cs

```diff
@@ -0,0 +1,330 @@
+        private VisualElement _viewBase;
+        private VirtualizedTreeView _tree;
+        private ToolbarSearchField _searchField;
+        private Label _statusLabel;
+        private ListView _jitAsmListView;
+        private Label _selectedItemName;
+        private Button _refreshButton;
+        private CancellationTokenSource _initCtes;
+        private CancellationTokenSource _searchCTS;
+        private Dictionary<Assembly, Type[]> _assemblyTypes = new Dictionary<Assembly, Type[]>();
+        private List<string> _loadedSourceLines = new List<string>();
+
+        public static IEnumerable<Type> SafeTypeLoad(Assembly asm)
+        {
+            try
+            {
+                return asm.GetTypes();
+            }
+            catch (ReflectionTypeLoadException rtle)
+            {
+                return rtle.Types;
+            }
+        }
+        [MenuItem("Window/JIT Inspector View", isValidateFunction: false, priority: 8)]
+        public static void ShowExample()
+        {
+            var wnd = GetWindow<JitInspectorView>();
+            wnd.titleContent = new GUIContent("JITInspector");
+        }
+
+        public void CreateGUI()
+        {
+            TemplateContainer templateContainer = _jitInspectorTemplateAsset.Instantiate();
+            _viewBase = templateContainer.Children().First();
+            for (int i = 0; i < templateContainer.styleSheets.count; i++)
+                _viewBase.styleSheets.Add(templateContainer.styleSheets[i]);
+
+            rootVisualElement.Add(_viewBase);
+
+            s_assemblies = AppDomain.CurrentDomain.GetAssemblies()
+                .OrderBy(asm => asm.FullName)
+                .ToList();
+
+            SetupUI();
+
+            EditorApplication.delayCall += InitializeAsync;
+        }
+        private void SetupUI()
+        {
+            _tree = new VirtualizedTreeView();
+            _tree.style.flexGrow = 1;
+            _viewBase.Q<VisualElement>("tree-container").Add(_tree);
+
+            _searchField = _viewBase.Q<ToolbarSearchField>("target-filter");
+            _searchField.RegisterValueChangedCallback(OnSearchChanged);
+            _statusLabel = new Label("Building index...");
+
+            _jitAsmListView = _viewBase.Q<ListView>("jit-asm");
+            _jitAsmListView.itemsSource = _loadedSourceLines;
+            _jitAsmListView.makeItem = () =>
+            {
+                Label label = new Label() { enableRichText = true };
+                label.AddToClassList("monospace");
+                return label;
+            };
+            _jitAsmListView.bindItem = (ve, i) => ((Label)ve).text = _loadedSourceLines[i];
+            _jitAsmListView.Rebuild();
+            _jitAsmListView.ScrollToItem(0);
+
+            _selectedItemName = _viewBase.Q<Label>("selected-item-name");
+
+            _tree.OnItemSelected += OnTreeItemSelected;
+            _tree.OnItemExpanded += OnTreeItemExpanded;
+
+            Refresh();
+        }
```

Comment:
I don't follow on this one

Reply:
The GitHub UI doesn't make it very clear. It's just about adding a blank line between functions i.e.
```cs
void M1()
{
}

void M2()
{
}
```
instead of
```cs
void M1()
{
}
void M2()
{
}
```